kube-ui/kube-ui-rc.yaml

```diff
@@ -0,0 +1,26 @@
+apiVersion: v1
+kind: ReplicationController
+metadata:
+  name: kube-ui-v1
+  namespace: default
+  labels:
+    k8s-app: kube-ui
+    version: v1
+    kubernetes.io/cluster-service: "true"
```

Comment:
@timstclair yes I'd agree. I'm not sure why @bcbroussard put them in this PR. @bcbroussard I'd say remove that from here.